src/System.Data.SQLite/ModuleInitializer.cs

```diff
@@ -0,0 +1,46 @@
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Data.SQLite
+{
+#if NET5_0
```

Comment:
Did we determine that `_OR_GREATER` isn't supported by our toolchain?

Reply:
Yes, unfortunately, `_OR_GREATER` was not supported by the toolchain.